pkgs/by-name/un/unbook/package.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, makeWrapper
+, calibre
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "unbook";
+  version = "0.7.3";
+
+  src = fetchFromGitHub {
+    owner = "ludios";
+    repo = "unbook";
+    rev = version;
+    hash = "sha256-KYnSIT/zIrbDFRWIaQRto0sPPmpJC8V7f00j4t/AsGQ=";
+  };
+
+  cargoHash = "sha256-AjyeTFgjl3XLplo8w9jne5FyKd2EciwbAKKiaDshpcA=";
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  postInstall = ''
+    wrapProgram $out/bin/unbook --prefix PATH : ${lib.makeBinPath [ calibre ]}
+  '';
+
+  meta = with lib; {
+    description = "Ebook → self-contained HTML; an {epub, mobi, azw, azw3, lit, chm} converter that outputs a single HTML file with images inlined as data: URIs";
```

```suggestion
    description = "An ebook to self-contained-HTML converter";
```

You can pick the first paragraph of website and use it as `longDescription` if you deem it necessary.

---

No, that looks perfect! Thanks